apps/elf-council-frontend/src/ui/gsc/useVotingPowerByDelegates.ts

```diff
@@ -1,55 +1,69 @@
-import { BigNumber, Event } from "ethers";
-import { useSmartContractEvents } from "@elementfi/react-query-typechain/src/hooks/useSmartContractEvents/useSmartContractEvents";
+import { BigNumber } from "ethers";
 import { lockingVaultContract, vestingContract } from "src/elf/contracts";
-import { useMemo } from "react";
 import { getFromBlock } from "src/elf-council-addresses/getFromBlock";
+import { useQuery, UseQueryResult } from "react-query";
+
+const oneAddress =
+  "0x0000000000000000000000000000000000000000000000000000000000000001";
 
 export type VotePowerByDelegate = Record<string, BigNumber>;
 
-export function useVotingPowerByDelegates(): VotePowerByDelegate {
-  const { data: vestingVaultEvents = [] } = useSmartContractEvents(
-    vestingContract,
-    "VoteChange",
-    {
-      fromBlock: getFromBlock(),
-    },
+/**
+ * Returns a record of all delegates and their respective voting power
+ * @returns {Promise<VotePowerByDelegate>} a mapping between delegate address -> voting power
+ */
+async function getVotingPowerByDelegates(): Promise<VotePowerByDelegate> {
```

Comment:
we have to query directly using query filters because this gives us a typed event back. For some reason events from the vesting vault have a switched `to` and `from` parameters in the array index so we need the typed interface instead.